Light.json

```diff
@@ -55967,6 +55914,339 @@
+      "front": {
+        "title": "Our Only Hope (V)",
+        "imageUrl": "https://res.starwarsccg.org/cards/Virtual14-Light/large/ouronlyhopev.gif",
+        "type": "Interrupt",
+        "subType": "Used",
+        "destiny": "4",
+        "gametext": "Relocate Prophecy Of The Force to a site. OR If He Is The Chosen One or He Will Bring Balance on table, take Yoda's Hut or a Death Star II site into hand from Reserve Deck; reshuffle."
+      },
+      "legacy": false
+    },	
+    {
+      "id": 6867,
+	  "gempId": "214_13",
+      "side": "Light",
+      "rarity": "PM",
+      "set": "214",
+      "printings": [
+        {
+          "set": "214"
+        }
+      ],
+      "front": {
+        "title": "Alter & Friendly Fire (V)",
+        "imageUrl": "https://res.starwarsccg.org/cards/Virtual14-Light/large/alterfriendlyfirev.gif",
+        "type": "Interrupt",
+        "subType": "Used Or Lost",
+        "destiny": "4",
+        "gametext": "This Interrupt is immune to opponent's Objective and does not cause you to lose Force to There Is No Try. USED: Cancel Sense. LOST: Target an Effect, Political Effect, or Utinni Effect, and one of your characters on table. Draw destiny. If destiny < character's ability, target Effect is canceled. OR If a battle was just initiated at a site where opponent has four or more characters, draw destiny. If destiny < number of opponent's characters at that site, opponent chooses one to be lost."
+      },
+        "canceledBy": [
+        "Control",
+        "Control & Set For Stun",
+        "Control (EP1)",
+        "Sense",
+        "Sense (EP1)"
+      ],
+      "cancels": [
+        "One Effect"
+      ],
+      "legacy": false
+    },		
+   {
+      "id": 6868,
+	  "gempId": "214_19",
+      "side": "Light",
+      "rarity": "R",
+      "set": "Endor",
+      "front": {
+        "title": "•Endor: Chief Chirpa's Hut (V)",
+        "imageUrl": "https://res.starwarsccg.org/cards/Virtual14-Light/large/endorchiefchirpashutv.gif",
+        "type": "Location",
+        "subType": "Site",
+        "uniqueness": "*",
+        "destiny": "0",
+        "icons": [
+          "Interior",
+          "Planet"
+        ],
+        "lightSideIcons": 2,
+        "darkSideIcons": 0
+,
+        "gametext": "Light: You may not generate more than 2 Force at non-battlegrounds. Ewoks deploy -2 here.  Dark: Force generation here may not be prevented by Objectives."
+      },
+      "pulledBy": [
+        "Wuta"
+      ],
+      "legacy": false
+    },	
+	
+	
+    {
+      "id": 6869,
+	  "gempId": "214_14",
+      "side": "Light",
+      "rarity": "C2",
+      "set": "214",
+      "printings": [
+        {
+          "set": "214"
+        }
+      ],
+      "front": {
+        "title": "ARC-170 Starfighter",
+        "imageUrl": "https://res.starwarsccg.org/cards/Virtual14-Light/large/arc170starfighter.gif",
+        "type": "Starship",
+        "subType": "Starfighter: Reconnaissance Fighter",
+        "uniqueness": "",
```

Comment:
I don't think non-unique characters/ships are supposed to have a uniqueness entry.